exercises/simple-cipher/canonical-data.json

```diff
@@ -0,0 +1,123 @@
+{
+  "exercise": "simple-cipher",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Random key cipher",
+      "property": "random",
+      "cases": [
+        {
+          "description": "It has a key made of letters",
+          "property": "key",
+          "input": "any",
+          "matches_regex": "^[a-z]+$"
+        },
+        {
+          "description": "It can encode",
+          "property": "encode",
+          "input_plaintext": "aaaaaaaaaa",
+          "output_key": "cipher.key"
+        },
+        {
+          "description": "It can decode",
+          "property": "decode",
+          "input_key": "cipher.key",
+          "output_string": "aaaaaaaaaa"
+        },
+        {
+          "description": "It is reversible",
+          "property": "encode -> decode",
+          "input_encode": "plaintext string",
+          "output_decode": "plaintext string"
+        }
+      ]
+    },
+    {
+      "description": "Incorrect key cipher",
+      "property": "incorrect",
```

Comment:
I'm not sure, but I don't think this line actually does anything. The `property` key is normally used for specifying which method/property to test. As this is just a container for individual cases, I think it can be removed.